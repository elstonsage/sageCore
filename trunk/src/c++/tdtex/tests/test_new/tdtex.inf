TDTEX -- 15 Jun 2006 14:54:59 -- [S.A.G.E. v5.2.0; 15 Jun 2006]
COPYRIGHT (C) 2005 CASE WESTERN RESERVE UNIVERSITY


Family structure information on the first 10 individuals read from file: ped

     PED. ID       IND. ID       SEX       PARENT1       PARENT2     
     ------------  ------------  --------  ------------  ------------
     0             1             M         0             0           
     0             2             F         0             0           
     0             3             M         1             2           


Phenotypes for the first 10 individuals read from file: ped

     PED. ID       IND. ID       aff                 
     ------------  ------------  --------------------
     0             1             1                   
     0             2             1                   
     0             3             1                   


Markers for the first 10 individuals read from file: ped

     PED. ID       IND. ID       m1          
     ------------  ------------  ------------
     0             1             A/B         
     0             2             A/B         
     0             3             A/B         

