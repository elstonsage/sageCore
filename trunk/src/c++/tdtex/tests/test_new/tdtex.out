TDTEX -- 15 Jun 2006 14:54:59 -- [S.A.G.E. v5.2.0; 15 Jun 2006]
COPYRIGHT (C) 2005 CASE WESTERN RESERVE UNIVERSITY
=====================

     Analysis #1

=====================

  Transmission type:        Allele             
  Marker:                   m1                 
  Trait:                    aff                
  Parental trait:           Not specified      
  Parental sex(es) scored:  Paternal/maternal  
  Max. children/family:                     1  
  Max. sib pairs/family:                    0  

----------------
  T/NT  A   B   
----------------
  A      0   1  
  B      1   0  

---------------------------------------------------------------------
  Group               Informative count  Total count  % Informative  
---------------------------------------------------------------------
  Pedigrees                           1            1   1.000000      
  Families                            1            1   1.000000      
  Affected children                   1            1   1.000000      
  Affected sib pairs                  0            0   0.000000      
  Sample size                         2            2   1.000000      

----------------------------------------------------------
  Exact test statistic              P-value    Std. err.  
----------------------------------------------------------
  Exact McNemar test                (skipped)             
  Monte Carlo McNemar test          (skipped)             
  Monte Carlo Marginal Homogeneity  (skipped)             

----------------------------------------------------
  Asymptotic test statistic          P-value        
----------------------------------------------------
  McNemar test                       1.00000000     
  Continuity corrected McNemar test  0.47950012     
  Marginal homogeneity test          1.00000000     

  Transmission type:        Allele         
  Marker:                   m1             
  Trait:                    aff            
  Parental trait:           Not specified  
  Parental sex(es) scored:  Paternal       
  Max. children/family:                 1  
  Max. sib pairs/family:                0  

---------------------------------------------------------------------
  Group               Informative count  Total count  % Informative  
---------------------------------------------------------------------
  Pedigrees                           0            1   0.000000      
  Families                            0            1   0.000000      
  Affected children                   0            0   0.000000      
  Affected sib pairs                  0            0   0.000000      
  Sample size                         0            0   0.000000      

----------------------------------------------------------
  Exact test statistic              P-value    Std. err.  
----------------------------------------------------------
  Exact McNemar test                (skipped)             
  Monte Carlo McNemar test          (skipped)             
  Monte Carlo Marginal Homogeneity  (skipped)             

----------------------------------------------------
  Asymptotic test statistic          P-value        
----------------------------------------------------
  McNemar test                       ----------     
  Continuity corrected McNemar test  ----------     
  Marginal homogeneity test          ----------     

  Transmission type:        Allele         
  Marker:                   m1             
  Trait:                    aff            
  Parental trait:           Not specified  
  Parental sex(es) scored:  Maternal       
  Max. children/family:                 1  
  Max. sib pairs/family:                0  

---------------------------------------------------------------------
  Group               Informative count  Total count  % Informative  
---------------------------------------------------------------------
  Pedigrees                           0            1   0.000000      
  Families                            0            1   0.000000      
  Affected children                   0            0   0.000000      
  Affected sib pairs                  0            0   0.000000      
  Sample size                         0            0   0.000000      

----------------------------------------------------------
  Exact test statistic              P-value    Std. err.  
----------------------------------------------------------
  Exact McNemar test                (skipped)             
  Monte Carlo McNemar test          (skipped)             
  Monte Carlo Marginal Homogeneity  (skipped)             

----------------------------------------------------
  Asymptotic test statistic          P-value        
----------------------------------------------------
  McNemar test                       ----------     
  Continuity corrected McNemar test  ----------     
  Marginal homogeneity test          ----------     

